### ISSUE TITLE
Additional NVDA menu

### ISSUE THREAD

**Issue by jacobchawala** [nvaccess/nvda#8685]
the feedback Hub menu should be added in the NVDA menu so that the user can easily give his/her feedback on NVDA issues without being necessitated to come in this website of Github. Microsoft has the feedback hub menu and we are happy with that menu.

#### Other information about your system:
             Machine name: DESKTOP-R0NVNF4
               Machine Id: {F1EFE585-AA4A-485E-9FE9-4DC0FCCDACBF}
         Operating System: Windows 10 Pro 64-bit (10.0, Build 17134) (17134.rs4_release.180410-1804)
                 Language: English (Regional Setting: English)
      System Manufacturer: TOSHIBA
             System Model: Satellite C850-C035
                     BIOS: InsydeH2O Version 03.72.306.50 (type: BIOS)
                Processor: Intel(R) Core(TM) i3-2348M CPU @ 2.30GHz (4 CPUs), ~2.3GHz
                   Memory: 8192MB RAM
      Available OS Memory: 8074MB RAM
                Page File: 2837MB used, 7156MB available
              Windows Dir: C:\WINDOWS
          DirectX Version: DirectX 12
      
#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
yes, the issue is the same.

**Comment by DrSooom** [comment]
And who shall moderate this new channel? Note, that A Feedback not only contains bug reports and feature requests. And furthermore it causes language problems if it isn't clear that bug reports and feature requests must be written in English on GitHub.

Well, adding an item in the NVDA-Submenu "Help" called "Report an issue on GitHub", which will open https://github.com/nvaccess/nvda/issues/new in the web browser, would be appreciated – and quite easy to do. Or just forget this "/new" because searching before reporting might be better. ;)

**Comment by leonardder** [comment]
cc @qchristensen, I recall that there is a similar issue.

**Comment by Qchristensen** [comment]
There was issue #7745 which went into a little more detail (suggesting different languages and emailing etc as well).

I think at the end of the day, it is going to come back to that if people aren't comfortable using GitHub, they are likely best raising their issue in the Users email group, confirming that it IS an issue, and getting help submitting it that way.  I'm happy to submit issues for people for instance, and do from time to time.  The key is once you submit an issue (by whatever means), the developers need to have a way to contact you again in case they need more information or need someone affected by the issue to test a try build with a potential fix etc.

Having said all that, I'm not against a link to https://github.com/nvaccess/nvda/issues or to the user group from the help menu.

**Comment by Adriani90** [comment]
Yes I agree with @qchristensen in this regard. Having a Feedback hub like Microsoft has makes sense if you have a centralized Team which works on addressing Feedback and issue Quality. As this is an open source Project, everyone can test and comment on issues. I think many users expect the issues to go directly to NV Access. But this is not the aim of such a Feedback reporting tool. Everyone should have the oportunity to look into issues and search for duplicate issues. So I think a link to Github in the help menu and a link to the user list is the best Option. If People cannot provide Feedback in inglish, then translators for that specific language should take this over for the users.

**Comment by leonardder** [comment]
Duplicate of #7745
